backend-api/src/functions/asyncActiveSession/handlerDependencies.ts

```diff
@@ -2,15 +2,31 @@ import { Logger } from "../services/logging/logger";
 import { Logger as PowertoolsLogger } from "@aws-lambda-powertools/logger";
 import { MessageName, registeredLogs } from "./registeredLogs";
 import { ITokenService, TokenService } from "./tokenService/tokenService";
+import { IDecryptJwe, JweDecryptor } from "./jwe/jweDecryptor";
+import { KMSAdapter } from "../adapters/kmsAdapter";
+import { GcmDecryptor, IDecryptSymmetric } from "./jwe/gcmDecryptor";
+import { IDecryptAsymmetric, RsaDecryptor } from "./jwe/rsaDecryptor";
 
 export interface IAsyncActiveSessionDependencies {
   env: NodeJS.ProcessEnv;
   logger: () => Logger<MessageName>;
-  tokenService: () => ITokenService;
+  kmsAdapter: (keyId: string) => KMSAdapter;
+  gcmDecryptor: () => GcmDecryptor;
+  rsaDecryptor: (kmsAdapter: KMSAdapter) => IDecryptAsymmetric;
```

Comment:
do we need to pass through a kmsAdapter here? Is that something we can import directly within the class?

Reply:
No longer applicable :D 😥